pkg/core/names.go

```diff
@@ -28,6 +29,11 @@ const (
 	RootReconcilerPrefix = "root-reconciler"
 )
 
+var (
+	// RootSyncPermissionsPrefix is the prefix used for all ClusterRoleBindings granting access to Root Reconcilers
+	RootSyncPermissionsPrefix = fmt.Sprintf("%s:%s", configsync.RootSyncKind, RootReconcilerPrefix)
```

Comment:
nit: this can be a `const` since it is comprised of two other consts

Reply:
It can't, at least not if we create it with `fmt.Sprintf` (my compiler complained that it wasn't a `const` expression).

Reply:
You should be able to do something like

```golang
const (
    RootSyncPermissionsPrefix = configsync.RootSyncKind + ":" + RootReconcilerPrefix
)
```

Reply:
Ah, yeah, that might work.